particle-iot/particle-cli
# bugfix/doctor

## Description

running  `particle device doctor` results in an error:

```
Cannot read property 'then' of undefined
```


## How to Test

1. Connect a `photon`, `electron`, or `p1` device via usb
2. Run `particle device doctor`
3. follow the on-screen instructions

Your device should be restored successfully.

## Related Issues / Discussions

https://community.particle.io/t/cli-1-44-0-device-doctor-error/51959
https://community.particle.io/t/device-doctor-error/51952


## Completeness

✅
